Makefile

```diff
@@ -305,9 +306,9 @@ update_canonical_urls:
 
 ## Tags and builds a release from start to finish.
 release: release-prereqs
-	$(MAKE) release-tag
-	$(MAKE) release-build
-	$(MAKE) release-verify
+	$(MAKE) RELEASE_CHART=true release-tag
```

Comment:
What is the purpose of `RELEASE_CHART`? Is it similar to the `CONFIRM` var we use on other projects?

Reply:
From line 67: 
```
# the presence of RELEASE_CHART indicates we're trying to cut an official chart release.
```

Reply:
OK. are you sure you don't want to use `CONFIRM=true` as it's more common across our makefile machinery?

Reply:
I don't think this is the same thing as CONFIRM. This is "do we want to publish an official helm chart", not "are you sue you want to make a release"